pkg/updater/repository/link_windows.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package repository
+
+import (
+	"fmt"
+)
+
+func linkRead(_ string) (string, error) {
+	return "", fmt.Errorf("not supported on windows")
+}
+
+func linkExists(_ string) (bool, error) {
+	return false, fmt.Errorf("not supported on windows")
+}
+
+func linkSet(_ string, _ string) error {
+	return fmt.Errorf("not supported on windows")
+}
+
+func linkDelete(_ string) error {
+	return fmt.Errorf("not supported on windows")
+}
```

Comment:
You might get a lint error by using `fmt.Errorf` without any format, you should use `errors.New` instead

Reply:
Lint looks like it's passing but happy to swap if that's preferred (even if it feels weird to use two package depending on wether I need the formatting or not if I'm being honest)

Reply:
Done ✔️